omero/sysadmins/version-requirements.rst

```diff
@@ -882,6 +663,22 @@ OMERO support policies
       - |Recommended|
       - |Recommended|
       - `Reference <https://access.redhat.com/articles/1299013>`__
+    * - 12
+      - from Sep 2018
+      - to Oct 2024
+      - |Unsupported|
+      - |Unsupported|
+      - |Supported|
+      - |Recommended|
```

Comment:
```suggestion
      - |Supported|
```
12 isn't an LTS: https://www.oracle.com/technetwork/java/java-se-support-roadmap.html

Reply:
Neither is 13. That's why I omitted them. Perhaps a different type of "unsupported/" could cover them.

Reply:
Maybe change this to `Unsupported` then? We don't need to go into details, and we're also not saying it's broken.